build-system/tasks/validator.js

```diff
@@ -43,4 +43,12 @@ module.exports = {
 };
 
 validator.description = 'Builds and tests the AMP validator.';
+validator.flags = {
+  'update_tests': 'Updates validation test output files',
```

Comment:
Add two spaces between `'` and `Updates` to make the output of `gulp help` more readable.